src/test/scala/com/cloudera/sparkts/models/ARIMAXSuite.scala

```diff
@@ -0,0 +1,136 @@
+/**
```

Comment:
I checked coefficients using R, our and pyflux libraries.
Coefficients are quite similar. You can see, that pyflux doesnt work in the same way like R. We generate better coeff than pyflux.
 
R: ar1=0.6901  ma1=-0.9328   x1=-3.2513   x2=0.1805  x3=0.0607   x4=-7.9104
Our: {u'ar': [0.7331745545680962], u'c': 0.003380661917224556,  u'ma': [-0.9638139784105879],  u'xreg': [-3.312077729440245,  0.0692571988623542,   0.05650501047060423,  -3.078919944223733]} 
Pyflux:
AR(1)                                    0.707 
MA(1)                                    -0.9871  
Beta x1                           0.0455
Beta x2                             0.0006  
Beta x3                      -0.0001 
Betax4                         -0.0196